include/deal.II/lac/petsc_precondition.h

```diff
@@ -378,7 +380,7 @@ namespace PETScWrappers
 
   /**
    * A class that implements the interface to use the PETSc Eisenstat
-   * preconditioner.
+   * preconditioner, which implements SSOR on each processor.
```

Comment:
Hm, is this what Eisenstat is? SSOR on each processor? (To be anally correct, it should probably be "...SSOR on the diagonal block owned by each processor".)